### PR TITLE
Change cumulative memory unit to be more readable

### LINKED ISSUE #4583: Cumulative Memory in UI should be converted to appropriate unit
Rather than always showing `GB`, it should be rounded to `TB`, `PB`, etc.

Example:

```
1,030,350.798GB seconds
```

### PR DESCRIPTION
**Pull request by prateek1306**
Fixes #4583

### CODE REVIEW

File: presto-main/src/main/resources/webapp/query.html [hunk 1]
```diff
@@ -297,7 +297,20 @@
 }
 
 function formatCumulativeMemory(cumulativeMemory) {
-    return (cumulativeMemory / Math.pow(1000.0, 4)).toLocaleString() + 'GB seconds';
+    unit = 'GB seconds';
```
Comment (martint) [inline presto-main/src/main/resources/webapp/query.html]: There's a formatDataSize function available. Use that instead of rolling your own here.

**Comment by prateek1306** [comment]
Changes done.

**Comment by martint** [comment]
@raghavsethi can you review?

File: presto-main/src/main/resources/webapp/query.html [hunk 2]
```diff
@@ -297,7 +297,9 @@
 }
 
 function formatCumulativeMemory(cumulativeMemory) {
-    return (cumulativeMemory / Math.pow(1000.0, 4)).toLocaleString() + 'GB seconds';
+    value = formatDataSize(cumulativeMemory/1000);
+    // formatDataSize would return 10M for 10 megabytes. Need to append string to make it 10MB seconds
```
Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/query.html]: Remove the comment, it's not required.

**Comment by raghavsethi** [comment]
@prateek1306 Change the commit message to "Change cumulative memory unit to be more readable", remove the comment, and I'll approve it.

**Comment by prateek1306** [comment]
@raghavsethi have pushed the changes.

**Comment by raghavsethi** [comment]
Manually merged into new version of Web UI. Thanks!

**Pull request closed without merging**